app/discord/deployCommands.server.ts

```diff
@@ -155,6 +155,9 @@ export const deployCommandsToGuild = async (guild: Guild) => {
                 ("description" in x ? x.description === c.description : true) &&
                 c.options?.every((o) =>
                   x.options?.some((o2) => o.name === o2.name),
+                ) &&
+                x.options?.every((o) =>
```

Comment:
Similarly as above - wouldn't it be more readable to properly name these variables and parameters according to their meaning?

Reply:
Honestly it's a bit of a nightmare — to improve readability I'd probably do a whole refactor into some helpers, short of doing that I'm okay just leaving it a little garbage for now